src/geovista/core.py

```diff
@@ -90,8 +78,13 @@ class SliceBias(Enum):
     """Enumerate meridian slice bias."""
 
     WEST = -1
+    """Preference for a slice to bias cells west of the chosen meridian."""
```

Comment:
This seems to document just fine 👍. I'll be using for more enumerations in future.

![image](https://github.com/bjlittle/geovista/assets/40734014/bae5b274-4172-41aa-95e8-755c12bbc484)